libvast/src/schema.cpp

```diff
@@ -194,4 +194,36 @@ caf::expected<schema> load_schema_file(const path& sf) {
   return to<schema>(*str);
 }
 
+caf::expected<schema>
+load_schema_dirs(const std::vector<std::string>& schema_paths) {
+  vast::schema types;
+  for (const auto& d : schema_paths) {
```

Comment:
```suggestion
  for (const auto& dir : schema_paths) {
```

In my eyes, `d` is not a common idiomatic variable name.